docs/source/gdpr/i18n/ultimate_gdpr.pt.rst

```diff
@@ -0,0 +1,50 @@
+.. _chap_consent_ultimate_gdpr_pt:
+
+Portuguese
+----------
+(translation courtesy of Ana Pina Rodrigues, Alexandre Sayal, Bruno Direito, Fernando Ferreira-Santos)
+
+**Version:** OBC-GDPR-ULT.pt 1.0.0
```

Comment:
please make sure you were translating the most recent version of the "ultimate" which already has version 

```
docs/source/gdpr/ultimate_gdpr.rst:**Version:** OBC-GDPR-ULT 1.1.0
```

Reply:
Added this to @con/portuguese -- will help any additional reviewers understand the process.

Reply:
> please make sure you were translating the most recent version of the "ultimate" which already has version
> 
> ```
> docs/source/gdpr/ultimate_gdpr.rst:**Version:** OBC-GDPR-ULT 1.1.0
> ```

Thank you for this, we translated version 1.1.0. I have now corrected this typo.